From a31c1395ab65522443bd3941539aefd503cabc59 Mon Sep 17 00:00:00 2001
From: Khanh Tran <khanh.tran.te@renesas.com>
Date: Mon, 28 Jun 2021 20:58:25 -0400
Subject: [PATCH] Remove nested structs in header

---
 lib/include/openamp/virtqueue.h | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/lib/include/openamp/virtqueue.h b/lib/include/openamp/virtqueue.h
index 5114fb2..ac37290 100644
--- a/lib/include/openamp/virtqueue.h
+++ b/lib/include/openamp/virtqueue.h
@@ -57,6 +57,11 @@ struct virtqueue_buf {
 	int len;
 };
 
+struct vq_desc_extra {
+	void *cookie;
+	uint16_t ndescs;
+};
+
 struct virtqueue {
 	struct virtio_device *vq_dev;
 	const char *vq_name;
@@ -99,10 +104,7 @@ struct virtqueue {
 	 * Other fields in this structure are not used currently.
 	 */
 
-	struct vq_desc_extra {
-		void *cookie;
-		uint16_t ndescs;
-	} vq_descx[0];
+	struct vq_desc_extra vq_descx[0];
 };
 
 /* struct to hold vring specific information */
-- 
2.7.4

