Index: git/edge_ac.c
===================================================================
--- git.orig/edge_ac.c
+++ git/edge_ac.c
@@ -31,8 +31,8 @@ static struct edgx_ifdesc *_ifm;
 
 int edgx_probe_ac(edgx_io_t *iobase, struct device *dev)
 {
-	int                i;
-	struct edgx_if    *itf;
+	int		   i;
+	struct edgx_if	  *itf;
 	struct edgx_ifreq  selfreq = { .id = AC_AC_ID, .v_maj = 1 };
 
 	if (!iobase)
@@ -42,8 +42,8 @@ int edgx_probe_ac(edgx_io_t *iobase, str
 	if (!_ifm)
 		return -ENOMEM;
 
-	edgx_info("Enumerating interfaces on device '%s %s'...\n",
-		  dev_driver_string(dev), dev_name(dev));
+	edgx_info("Enumerating interfaces on device '%s %s' %px...\n",
+		  dev_driver_string(dev), dev_name(dev), iobase);
 	for (i = 0, itf = (struct edgx_if *)(((u8 *)iobase) + AC_IFMAP_OFS);
 	     (itf->id != AC_EOT_ID) && (i < AC_MAX_ENTRIES);
 	     itf++, i++) {
@@ -78,7 +78,7 @@ const struct edgx_ifdesc *edgx_ac_get_if
 	for (i = 0; i < AC_MAX_ENTRIES; i++) {
 		struct edgx_ifdesc *itf = &_ifm[i];
 
-		if (itf->id        == ifreq->id &&
+		if (itf->id	   == ifreq->id &&
 		    itf->ver.v_maj == ifreq->v_maj)
 			return itf;
 	}
@@ -92,12 +92,12 @@ const struct edgx_ifdesc *edgx_ac_if2pti
 	if (!ifdesc || !ptifdesc || (!(ifdesc->ptmap & BIT(pt))))
 		return NULL;
 
-	ptifdesc->id      = ifdesc->id;
-	ptifdesc->ver     = ifdesc->ver;
+	ptifdesc->id	  = ifdesc->id;
+	ptifdesc->ver	  = ifdesc->ver;
 	/* TODO: Change to do_div! */
-	ptifdesc->len     = ifdesc->len / hweight_long(ifdesc->ptmap);
+	ptifdesc->len	  = ifdesc->len / hweight_long(ifdesc->ptmap);
 	ptifdesc->iobase  = ifdesc->iobase + (pt * ptifdesc->len);
-	ptifdesc->ptmap   = BIT(pt);
+	ptifdesc->ptmap	  = BIT(pt);
 
 	return ptifdesc;
 }
Index: git/edge_bridge.c
===================================================================
--- git.orig/edge_bridge.c
+++ git/edge_bridge.c
@@ -48,37 +48,37 @@
 struct edgx_br {
 	struct {
 		struct net_device *dev;
-		unsigned int       refs;
+		unsigned int	   refs;
 	} bridge;
 	struct {
-		u8           base_mac[ETH_ALEN];
+		u8	     base_mac[ETH_ALEN];
 		unsigned int id;
 	} hw;
 
 	struct list_head     entry;
 	struct list_head     grp_list;
-	struct device       *pdev;
+	struct device	    *pdev;
 
 	struct edgx_time    *time;
-	ptvec_t              time_map;
+	ptvec_t		     time_map;
 
 	struct edgx_stat    *stat;
-	ptvec_t              stat_map;
+	ptvec_t		     stat_map;
 
-	struct edgx_com     *com;
+	struct edgx_com	    *com;
 
-	edgx_io_t           *brmgmt;
-	edgx_io_t           *brparam;
+	edgx_io_t	    *brmgmt;
+	edgx_io_t	    *brparam;
 	struct edgx_brfdb   *fdb;
 	struct edgx_br_vlan *vlan;
 
-	struct edgx_pt      *br_pt[EDGX_BR_MAX_PORTS];
-	struct edgx_pt      *ep_pt;
+	struct edgx_pt	    *br_pt[EDGX_BR_MAX_PORTS];
+	struct edgx_pt	    *ep_pt;
 
-	edgx_io_t           *iobase;
+	edgx_io_t	    *iobase;
 
-	clock_t              ageing_time;
-	unsigned int         cycle_ns;
+	clock_t		     ageing_time;
+	unsigned int	     cycle_ns;
 
 	struct edgx_st_com	*st_com;
 	struct edgx_sid_br	*sid;
@@ -91,12 +91,12 @@ struct edgx_br {
 	struct edgx_br_irq	*irq;
 
 	enum edgx_lnk_mode	lnk_mode;
-	const char          *driver;
+	const char	    *driver;
 };
 
 struct _sysfs_grp {
 	struct attribute_group *grp;
-	struct list_head        list;
+	struct list_head	list;
 };
 
 static char *netif = "(none)";
@@ -637,7 +637,7 @@ static int _edgx_br_get_delays(struct ed
 	/* delays->ingress_port = idx /
 	 * .........(sizeof(struct edgx_delay_tc) * (EDGX_BR_MAX_PORTS - 1));
 	 * help_calc = idx %
-	 *          (sizeof(struct edgx_delay_tc) * (EDGX_BR_MAX_PORTS - 1));
+	 *	    (sizeof(struct edgx_delay_tc) * (EDGX_BR_MAX_PORTS - 1));
 	 */
 	rem = do_div(help_idx,
 		     sizeof(struct edgx_delay_tc) * (EDGX_BR_MAX_PORTS - 1));
@@ -755,15 +755,15 @@ static ssize_t delays_read(struct file *
 }
 
 /* Bridge Attributes */
-EDGX_DEV_FCT_ATTR_RO(br_type,   type,            "BridgeType");
-EDGX_DEV_FCT_ATTR_RO(br_nports, num_ports,       "BridgeNumPorts");
-EDGX_DEV_FCT_ATTR_RO(br_comps,  edgx_sysfs_true, "BridgeComponents");
-EDGX_DEV_FCT_ATTR_RO(br_tc_en,  edgx_sysfs_true, "BridgeTrafficClassesEnabled");
+EDGX_DEV_FCT_ATTR_RO(br_type,	type,		 "BridgeType");
+EDGX_DEV_FCT_ATTR_RO(br_nports, num_ports,	 "BridgeNumPorts");
+EDGX_DEV_FCT_ATTR_RO(br_comps,	edgx_sysfs_true, "BridgeComponents");
+EDGX_DEV_FCT_ATTR_RO(br_tc_en,	edgx_sysfs_true, "BridgeTrafficClassesEnabled");
 
 /* Component Attributes */
-EDGX_DEV_FCT_ATTR_RO(comp_id,     edgx_sysfs_one, "ComponentId");
-EDGX_DEV_FCT_ATTR_RO(comp_type,   type,           "ComponentType");
-EDGX_DEV_FCT_ATTR_RO(comp_nports, num_ports,      "ComponentNumPorts");
+EDGX_DEV_FCT_ATTR_RO(comp_id,	  edgx_sysfs_one, "ComponentId");
+EDGX_DEV_FCT_ATTR_RO(comp_type,	  type,		  "ComponentType");
+EDGX_DEV_FCT_ATTR_RO(comp_nports, num_ports,	  "ComponentNumPorts");
 
 /* FDB Attributes */
 EDGX_DEV_ATTR_RO(fdb_sz,      "FdbSize");
@@ -771,12 +771,12 @@ EDGX_DEV_ATTR_RO(fdb_smac_sz, "FdbStatic
 
 /* Capability Attributes */
 EDGX_DEV_FCT_ATTR_RO(efs,     edgx_sysfs_false, "ExtendedFilteringServices");
-EDGX_DEV_FCT_ATTR_RO(tc,      edgx_sysfs_true,  "TrafficClasses");
+EDGX_DEV_FCT_ATTR_RO(tc,      edgx_sysfs_true,	"TrafficClasses");
 EDGX_DEV_FCT_ATTR_RO(seip,    edgx_sysfs_false, "StaticEntryIndividualPort");
-EDGX_DEV_FCT_ATTR_RO(ivl,     edgx_sysfs_true,  "IVLCapable");
-EDGX_DEV_FCT_ATTR_RO(svl,     edgx_sysfs_true,  "SVLCapable");
-EDGX_DEV_FCT_ATTR_RO(hybrid,  edgx_sysfs_true,  "HybridCapable");
-EDGX_DEV_FCT_ATTR_RO(pvidtag, edgx_sysfs_true,  "ConfigurablePvidTagging");
+EDGX_DEV_FCT_ATTR_RO(ivl,     edgx_sysfs_true,	"IVLCapable");
+EDGX_DEV_FCT_ATTR_RO(svl,     edgx_sysfs_true,	"SVLCapable");
+EDGX_DEV_FCT_ATTR_RO(hybrid,  edgx_sysfs_true,	"HybridCapable");
+EDGX_DEV_FCT_ATTR_RO(pvidtag, edgx_sysfs_true,	"ConfigurablePvidTagging");
 EDGX_DEV_FCT_ATTR_RO(lvlan,   edgx_sysfs_false, "LocalVlanCapable");
 
 /* Common attributes for bridge and every port */
@@ -827,8 +827,8 @@ static struct bin_attribute *ieee8021_br
 };
 
 static struct attribute_group ieee8021_bridge_group = {
-	.name      = "ieee8021Bridge",
-	.attrs     = ieee8021_bridge_attrs,
+	.name	   = "ieee8021Bridge",
+	.attrs	   = ieee8021_bridge_attrs,
 	.bin_attrs = ieee8021_bridge_binattrs,
 };
 
@@ -905,11 +905,11 @@ int edgx_br_probe_one(unsigned int br_id
 	if (!br)
 		return -ENOMEM;
 
-	br->iobase       = base;
-	br->pdev         = dev;
-	br->bridge.dev   = NULL;
-	br->bridge.refs  = 0;
-	br->hw.id        = br_id;
+	br->iobase	 = base;
+	br->pdev	 = dev;
+	br->bridge.dev	 = NULL;
+	br->bridge.refs	 = 0;
+	br->hw.id	 = br_id;
 	br->irq		 = irq;
 	br->driver   = drv_name;
 
Index: git/edge_platform.c
===================================================================
--- git.orig/edge_platform.c
+++ git/edge_platform.c
@@ -40,22 +40,19 @@
 #define PFX "EDGX-PLATFORM: "
 
 static const struct of_device_id flx_pio_dt_ids[] = {
-	{ .compatible = "flx,pio" },
-	{ /* sentinel */ },
+	{ .compatible = "flx,pio", },
+	{ /* sentinel */ }
 };
 
 static const struct of_device_id edgx_sw_dt_ids[] = {
-	{ .compatible = "ttt,deip-sw",
-	},
-	{ .compatible = "ttt,es",
-	},
-	{ /* sentinel */ },
+	{ .compatible = "ttt,deip-sw", },
+	{ .compatible = "ttt,es", },
+	{ /* sentinel */ }
 };
 
 static const struct of_device_id edgx_mdio_dt_ids[] = {
-	{ .compatible = "flx,eth-mdio",
-	},
-	{ /* sentinel */ },
+	{ .compatible = "flx,eth-mdio", },
+	{ /* sentinel */ }
 };
 
 static int bridge_id;
